File: spec/javascripts/sort_spec.js
describe("enochSort", function(){
  it("should sort an array of numbers", function() {
    var arry = [5, 6, 3, 2, 7, 4]
    expect(enochSort(arry)).toEqual([2, 3, 4, 5, 6, 7]);
  });
  it("should sort an array of numbers", function() {
    var arry = ["John", "Alex", "Stephanie", "Zinnia", "Alicia"];
    expect(enochSort(arry)).toEqual(["Alex", "Alicia", "John", "Stephanie", "Zinnia"])
  });
  it("should be able to handle double digit numbers", function() {
    var arry = [5, 6, 33, 2, 7, 4]
    expect(enochSort(arry)).toEqual([2, 4, 5, 6, 7, 33]);
  });
  it("should be able to handle duplicates", function() {
    var arry = [5, 6, 33, 2, 6, 7, 4]
    expect(enochSort(arry)).toEqual([2, 4, 5, 6, 6, 7, 33]);
  });
})

File: spec/sort_spec.rb
require 'spec_helper'
require_relative '../sort'

describe '#sort' do
  it "should sort numbers in ascending order" do
    array = [2,1].my_sort
    expect(array).to eq([1, 2])
  end

  it "should be able to sort more" do
    array = [3,4,2,1].my_sort
    expect(array).to eq([1, 2, 3, 4])
  end

  it 'should be able to sort even' do
    array = []
    100.times { array << rand(100) }
    expect(array.my_sort).to eq(array.sort)
  end

  
end
